sphinx/builders/linkcheck.py

```diff
@@ -330,7 +347,7 @@ class CheckResult(NamedTuple):
     uri: str
     docname: str
     lineno: int
```

Comment:
Shouldn't the type alias better be singular? The variable holds exactly one status.

```suggestion
    status: _Status | Literal['']
```